src/App.js

```diff
@@ -1,7 +1,39 @@
+import { Route, Routes } from "react-router-dom";
 import "./App.css";
+import Footer from "./Components/Footer";
+import Nav from "./Components/Navigation";
+
+import Landing from "./Pages/Landing";
+import Login from "./Pages/Login";
+import Signup from "./Pages/Signup";
+
+import Home from "./Components/Home";
+import Explore from "./Components/Explore";
+import Liked from "./Components/Liked";
+import Playlist from "./Components/Playlist";
+import WatchLater from "./Components//WatchLater";
+import History from "./Components/History";
```

Comment:
Instead of doing these imports specifically from each file. You can have an **index.js** file in **Components folder** and keep all your *exports there*. 

So that you can just **import** these components in a *single line*.

**Eg:**
 
 **/Components**
          - index.js
          
**Inside index.js**
              
         export {Header} from "./Header/Header
         Similarly, you can add exports of all components in this file

**Now in App.js:**
                  
                   import {Header , Footer} from "./Components"

Reply:
Hi, thank you for the suggestions. I've made the requested changes can you please check.